### PR TITLE
Adding the content from the Google doc to help.tpl.html

### PR DESCRIPTION
**Pull request by rzelayafavila**
Quick PR adding the content from the Google doc, which Jie and I communicated about a couple of days ago.

### CODE REVIEW

File: interface/src/app/help/help.tpl.html [hunk 1]
```diff
@@ -3,50 +3,48 @@
...
-    Gene content goes here. Lorem ipsum dolor sit amet, consectetur adipiscing
-    elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-    aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
-    voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
-    occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
-    anim id est laborum.
+    ADAGE model is built with genes in Pseudomonas aeruginosa PAO1 strain.
+    Information about genes are obtained from the following NCBI website:
+    ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Archaea_Bacteria/Pseudomonas_aeruginosa_PAO1.gene_info.gz
```
Comment (rzelayafavila) [inline interface/src/app/help/help.tpl.html]: Question @tj8901nm @cgreene : Should this be a hyperlink to this ftp site?

File: interface/src/app/help/help.tpl.html [hunk 2]
```diff
@@ -3,50 +3,48 @@
...
-    deserunt mollit anim id est laborum.
+    Experiments are obtained from the ArrayExpress database. Each experiment
+    has a “E-XXXX-number” accession number defined by ArrayExpress.
+  </div>
+  <br>
+
+  <div class="lead" id="Expression compendium">
+    <h3><u>Expression compendium?</u></h3>
+    The webserver is currently using P.a. expression compendium <!--(version
+    number here?) -->. It is built on July 31 2015 and contains 125 datasets
```
Comment (rzelayafavila) [inline interface/src/app/help/help.tpl.html]: @tj8901nm @cgreene : What version number do we want to put here?

Comment (cgreene) [inline interface/src/app/help/help.tpl.html]: I think you should refer to it by month day year as you do in the following sentence. We don't yet have the `data-refinery` which will give us more informative versions.

File: interface/src/app/help/help.tpl.html (hunk 1 above)
Comment (cgreene) [inline interface/src/app/help/help.tpl.html]: Would it be easier to say that information about genes is obtained from NCBI, where NCBI is a link to that URL?

File: interface/src/app/help/help.tpl.html [hunk 3]
```diff
@@ -3,50 +3,48 @@
...
   <div class="lead" id="Genes">
     <h3><u>Genes</u></h3>
-    Gene content goes here. Lorem ipsum dolor sit amet, consectetur adipiscing
-    elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-    aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
-    voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
-    occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
-    anim id est laborum.
+    ADAGE model is built with genes in Pseudomonas aeruginosa PAO1 strain.
```
Comment (tj8901nm) [inline interface/src/app/help/help.tpl.html]: "Pseudomonas aeruginosa" should be italicized.

File: interface/src/app/help/help.tpl.html [hunk 4]
```diff
@@ -3,50 +3,48 @@
...
   </div>
   <br>
 
+  <div class="lead" id="Signatures">
+    <h3><u>Signatures</u></h3>
+    Signatures are functional gene sets extracted by an ADAGE model. A
+    signature contains genes contributing extremely positive high or
+    negative high weights to a specific node. One node results in two
+    signatures on positive and negative sides. The positive and negative
+    signatures are treated independently.
```
Comment (dongbohu) [inline interface/src/app/help/help.tpl.html]: Is it necessary to sate explicitly that each signature is in the range of [-1.0, 1.0], or it is common sense?

Comment (tj8901nm) [inline interface/src/app/help/help.tpl.html]: I feel this is too detailed for a help page.

File: interface/src/app/help/help.tpl.html (hunk 3 above)
Comment (rzelayafavila) [inline interface/src/app/help/help.tpl.html]: Ok, I'll change that

File: interface/src/app/help/help.tpl.html (hunk 1 above)
Comment (rzelayafavila) [inline interface/src/app/help/help.tpl.html]: Ok - are we sure that is the URL that we want though? It seems like clicking on it (even if the browser is instructed to open a new tab/window), just downloads that gene info file. Is that what we want or do we want users to actually land on a web page in the browser?

File: interface/src/app/help/help.tpl.html (hunk 2 above)
Comment (rzelayafavila) [inline interface/src/app/help/help.tpl.html]: Ok, I'll remove the part about version number.

**Comment by rzelayafavila** [comment]
Ok, I made some changes. Also, I have a live instance of the Adage server running on my machine, so you can check out how the help page looks with the latest changes here: http://165.123.67.154:8000/#/help

**Comment by cgreene** [comment]
Assigning to @georgiadoing - she is working on a more biologist-friendly draft of this page. When she thinks the text is ready to merge, that's probably the time to bring it in. We can always revise later.

**Comment by rzelayafavila** [comment]
Hi @cgreene @tj8901nm @georgiadoing: I have added the content from the google doc to the adage help page. You can still visit it at http://165.123.67.154:8000/#/help . I am wondering what is the paper we are supposed to cite at the end of the third sentence in the **Nodes** section though - what is this citation supposed to be? If somebody has it, I can put it in.

**Comment by rzelayafavila** [comment]
Ok, I have included the full citation at the end of the page. @cgreene I followed the format we used when citing the Tribe paper in the webserver, and so I put in the year of the preprint (2016) in as well. Does that look ok?

**Comment by tj8901nm** [comment]
I think the citation used the old paper name. Since we are going to change the name once again, it might not worth worrying about it now.

I made some modifications in the google doc. Please take a look!

**Comment by rzelayafavila** [comment]
Ok, I have added the changes. Let me know how it looks!

**Comment by cgreene** [comment]
I am ok with this but @georgiadoing or @tj8901nm should mark if it's approved.

**Comment by rzelayafavila** [comment]
Ok, great, I will merge. Thanks!

**Pull request merged**